### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.servicefabric.models.chaos_restart_node_fault_scheduled_event_py3.ChaosRestartNodeFaultScheduledEvent
  fullName: azure.servicefabric.models.chaos_restart_node_fault_scheduled_event_py3
  langs:
  - python
  module: azure.servicefabric.models.chaos_restart_node_fault_scheduled_event_py3
  name: chaos_restart_node_fault_scheduled_event_py3
  source:
    id: chaos_restart_node_fault_scheduled_event_py3
    path: azure-servicefabric\azure\servicefabric\models\chaos_restart_node_fault_scheduled_event_py3.py
    remote:
      branch: master
      path: azure-servicefabric\azure\servicefabric\models\chaos_restart_node_fault_scheduled_event_py3.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.servicefabric.models.chaos_restart_node_fault_scheduled_event_py3
references:
- fullName: azure.servicefabric.models.chaos_restart_node_fault_scheduled_event_py3.ChaosRestartNodeFaultScheduledEvent
  isExternal: false
  name: ChaosRestartNodeFaultScheduledEvent
  parent: azure.servicefabric.models.chaos_restart_node_fault_scheduled_event_py3
  uid: azure.servicefabric.models.chaos_restart_node_fault_scheduled_event_py3.ChaosRestartNodeFaultScheduledEvent
